doc/api/stream.md

```diff
@@ -876,7 +876,7 @@ added: v8.0.0
 * `error` {Error} Error which will be passed as payload in `'error'` event
 * Returns: {this}
 
-Destroy the stream, and emit `'error'` and `'close'`. After this call, the
+Destroy the stream. Optionally emit an 'error' event, and always emit a 'close' event. After this call, the
```

Comment:
Our doc linting requires long lines to be wrapped at 80 characters maximum)

Reply:
@vsemozhetbyt hi, thanks for pointing this! I've fixed this line (and checked others as well).